packages/block-library/src/search/edit.js

```diff
@@ -446,33 +447,34 @@ export default function SearchEdit( {
 							value={ `${ width }${ widthUnit }` }
 							units={ units }
 						/>
-						<ButtonGroup
-							className="wp-block-search__components-button-group" // unused, kept for backwards compatibility
-							aria-label={ __( 'Percentage Width' ) }
+						<ToggleGroupControl
```

Comment:
Same as https://github.com/WordPress/gutenberg/pull/65346#discussion_r1760902661 re. adding the `isBlock` property

Reply:
Updated.